devicetypes/smartthings/zigbee-lock-without-codes.src/zigbee-lock-without-codes.groovy

```diff
@@ -160,8 +160,10 @@ def unlock() {
 def parse(String description) {
 	def result = null
 	if (description) {
-		if (description.startsWith('read attr -')) {
+		if (description?.startsWith('read attr -')) {
 			result = parseAttributeResponse(description)
+		} else if (description?.startsWith('zone report')) {
```

Comment:
Just one question. Isn't there any chance to come 'zone status'?
Then I think it will miss handling it.

Reply:
I haven't notice 'zone status' what I was in logs was only 'zone report'